### ISSUE TITLE
CODEOWNERS recognized as YAML

### ISSUE THREAD

**Issue by zm-cttae-archive** [microsoft/vscode#172573]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.74.3
- OS Version: 21H2

Steps to Reproduce:

1. Copy the following code:
```
- @M00TSDNS
```
2. Save it to `./.github/CODEOWNERS` file
3. It's recognised as YAML

Ideal fix: ~~map `CODEOWNERS` filename to Markdown.~~ it is a custom format and should maybe just be plain text

**Comment by TylerLeonhardt** [comment]
Workaround is to install a CODEOWNERS extension:

https://marketplace.visualstudio.com/items?itemName=jasonnutter.vscode-codeowners

But ideally we just don't guess YAML here

**Comment by M00TSDNS** [comment]
Oh brill there's an extension! (My bad for the "Markdown" ping lol)
I did ask a question or two on automatic lang detection around the time it was implemented.
IIRC "custom langs" came up but it was a very involved discussion and presumably nobody wanted that blocker :P

I really like the approach suggested in https://github.com/yoeo/guesslang/issues/45#issuecomment-928311323. Source code is super regular and repeats keywords or punctuation a lot, so unless we're dealing with a SQL, Pythonic or Lua like language, the model chain would be great.
